packages/kit-bg/src/services/ServiceFirmwareUpdate/ServiceFirmwareUpdate.ts

```
@@ -80,6 +80,7 @@ import type {
   IVersionArray,
 } from '@onekeyfe/hd-core';
 import type { Success } from '@onekeyfe/hd-transport';
+import { defaultLogger } from '../../../../shared/src/logger/logger';
```

ESLint

[error] Relative import from another package is not allowed. Use `@onekeyhq/shared/src/logger/logger` instead of `../../../../shared/src/logger/logger`

(import/no-relative-packages)